src/components/Library/LibraryBook/LibraryBook.tsx

```diff
@@ -33,7 +33,7 @@ const LibraryBook: React.FC<BookProps> = ({ book }) => {
           <span className={styles.subtitle}>{book.pages}</span>
         </div>
         {book.status === 'done' ? (
```

Comment:
maybe for 'done' and another status create some enum?

Reply:
Ok, good idea